cloud_connectors/azure/sample-dtdl/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure Digital Twins Setup
+
+This document descibes how to setup and Azure Digital Twins instance to use the sample DTDL in this folder.
+
+## Automated Azure Digital Twins Setup
+
+Before starting this section, please read [Prerequisites for Automated Deployment of Azure Resources](../README.md#prerequisites-for-automated-deployment-of-azure-resources).
+
+1. Sign in with Azure CLI. Follow the prompts after entering the following command.
+
+```shell
+az login --use-device-code
+```
+
+1. Deploy Azure Digital Twins to your resource group.
+
+```shell
+cd {repo-root}/cloud_connectors/azure/scripts
+chmod +x digital_twins_setup.sh
```

Comment:
It would be good to be explicit here.  I assume (and hope) that the user is the default for this case, so use "u+x" rather than "+x".  This is just a suggestion.